packages/sn-controls-react/package.json

```diff
@@ -44,7 +44,6 @@
     "@sensenet/pickers-react": "^1.0.2",
     "lodash.debounce": "^4.0.8",
     "material-ui-pickers": "^2.2.1",
-    "moment": "^2.24.0",
```

Comment:
date-io depends on moment. I'd let that here or add that as a peer dependency.

Reply:
ok